cmd/flags_node.go

```diff
@@ -62,13 +64,24 @@ func ParseNodeFlags(ctx context.Context, cmd *cobra.Command) (context.Context, e
 
 	nodeConfig := cmd.Flag(nodeConfigFlag).Value.String()
 	if nodeConfig != "" {
-		cfg, err := node.LoadConfig(nodeConfig)
+		// try to load config from given path
+		cfg, err := nodebuilder.LoadConfig(nodeConfig)
 		if err != nil {
 			return ctx, fmt.Errorf("cmd: while parsing '%s': %w", nodeConfigFlag, err)
 		}
 
-		ctx = WithNodeOptions(ctx, node.WithConfig(cfg))
+		ctx = WithNodeConfig(ctx, cfg)
+	} else {
+		// check if config already exists at the store path and load it
+		path := StorePath(ctx)
+		expanded, err := homedir.Expand(filepath.Clean(path))
+		if err != nil {
+			return ctx, err
+		}
+		cfg, err := nodebuilder.LoadConfig(filepath.Join(expanded, "config.toml"))
+		if err == nil {
+			ctx = WithNodeConfig(ctx, cfg)
+		}
```

Comment:
Why do we ignore the error here?

Reply:
Because this is an attempt to load an existing config if it exists, if not, we initialise with a new one.

Reply:
Ah, got it